tests/ttnn/unit_tests/test_multi_device_async.py

```diff
@@ -243,10 +243,14 @@ def test_multi_device_data_parallel_op_chain(pcie_mesh_device, program_cache, in
         pcie_mesh_device.get_device(device).enable_async(False)
 
 
-@pytest.mark.parametrize("layout", [ttnn.TILE_LAYOUT, ttnn.ROW_MAJOR_LAYOUT])
+@pytest.mark.parametrize(
+    "layout",
+    [
+        ttnn.ROW_MAJOR_LAYOUT,
```

Comment:
Why do we remove `TILE_LAYOUT` here?
Edit: I see that it only supports row-major below.